api/envoy/extensions/filters/http/rbac/v3/rbac.proto

```diff
@@ -34,6 +34,11 @@ message RBAC {
   config.rbac.v3.RBAC rules = 1
       [(udpa.annotations.field_migrate).oneof_promotion = "rules_specifier"];
 
+  // If specified, rules will emit stats with the given prefix.
+  // This is useful to distinguish the stat when there are more than 1 RBAC filter configured with
```

Comment:
I may have missed that, but is there a test that validates that 2 RBAC filters have a different prefix for their stats?